data/101/749/169/101749169.geojson

```diff
@@ -455,11 +455,12 @@
     "wd:wordcount":10613,
     "wof:belongsto":[
         102191581,
-        1159297743,
         85633121,
+        1159297743,
         85682593
     ],
     "wof:breaches":[],
+    "wof:capital_of":85682593,
```

Comment:
Do we need to adjust the property description?

- https://github.com/whosonfirst/whosonfirst-properties/blob/master/properties/wof/capital_of.json

We're storing wof ID ints not strings in the PRs (which I think is correct), but doesn't match the documentation?

Also... should these technically be lists? Or document that it stores the ID of the "highest" placetype (eg if capital of both country and region then the country ID would be listed), like for Washington DC or several other federal capital territories? Same thing sometimes happens with county and region capitals (like Sacramento)?

Reply:
Yeah, when we introduced the `wof:capital` and `wof:capital_of` properties, `wof:capital` was a list and `wof:capital_of` was a string. Further iterations changed `wof:capital_of` to an integer. The thinking at the time was the a country could have more than one capital, but capital cities only were only capital to a single country.

Now that we're fitting in admin1, it makes sense that both of these properties would be lists and documentation needs to be updated.

This would make things a bit easier, as a locality could be the capital of both a country and it's parent region. I'll add new commits to address this.

Reply:
Got it, thanks.